mlrun/projects/project.py

```diff
@@ -5266,6 +5266,12 @@ def _validate_file_path(self, file_path: str, param_name: str):
                 f"Invalid '{param_name}': '{file_path}'. Got a remote URL without a file suffix."
             )
 
+        _, in_context = self.get_item_absolute_path(file_path)
```

Comment:
this function may raise `FileNotFoundError`. how can that not be a regression? what guards us from falling into that exception?

Reply:
It raises file not found only when the file is an absolute path i.e. not in context. It would also raise `MLRunInvalidArgumentError` for the same reason below which is more elaborated.
I think we need to add a UT case on `test_set_workflow_path_validation` for absolute path that doesn't exist and make sure that the error is clear